eng/pipelines/common/createdump-entitlements.plist

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is only used by the pipeline yaml. In the dotnet-release repo, these files are in eng/pipelines/tools so I guess I thought it would be ok to put them under eng/pipelines and then "common" because both coreclr and installer use them.